resources/hab_build.rb

```diff
@@ -84,7 +84,9 @@ def last_build_env
     live_stream new_resource.live_stream
     sensitive true
   end
+end
 
+action :publish_env_pin do
```

Comment:
Can we please add some test coverage for this action to https://github.com/chef-cookbooks/habitat-build/blob/master/spec/unit/resources/hab_build_spec.rb